identity/handler.go

```diff
@@ -180,7 +180,7 @@ func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Para
 		isam[i] = WithCredentialsMetadataAndAdminMetadataInJSON(identity)
 	}
 
-	migrationpagination.PaginationHeader(w, urlx.AppendPaths(h.r.Config().SelfAdminURL(r.Context()), RouteCollection), total, page, itemsPerPage)
+	migrationpagination.PaginationHeader(w, urlx.AppendPaths(h.r.Config().SelfAdminURL(r.Context()), RouteCollection), total+int64(itemsPerPage), page, itemsPerPage)
```

Comment:
This is a super ugly hack, please suggest a better way if you know any. `page-1` does not work because then we get `1` as the next page for `1`.

Reply:
Why is this necessary? Doesn't that value come from the DB?

Reply:
This breaks the x-total-count unfortunately.